locations/method_locations_test.go

```diff
@@ -47,3 +47,22 @@ func TestMethodResponseType(t *testing.T) {
 		t.Errorf(diff)
 	}
 }
+
+func TestMethodHTTPRule(t *testing.T) {
+	f := parse(t, `
+		import "google/api/annotations.proto";
+		service Library {
+		  rpc GetBook(GetBookRequest) returns (Book) {
+		    option (google.api.http) = {
+		      get: "/v1/{name=publishers/*/books/*}"
+		    };
+		  }
+		}
+		message GetBookRequest{}
+		message Book {}
+	`)
+	loc := MethodHTTPRule(f.GetServices()[0].GetMethods()[0])
+	if diff := cmp.Diff(loc.GetSpan(), []int32{5, 4, 7, 6}); diff != "" {
```

Comment:
Can we clarify what the 5,4,7,6 is here? I suspect someone reading this might go "OK you check that the output is a magic number..."

Reply:
+1, maybe something like:
```golang
wantTextSpan := []int{
  5, // start line
  4, // start column
  7, // end line
  6, // end column
}
```

Reply:
Done.